superset/contrib/docker/docker-entrypoint.sh

```diff
@@ -18,7 +18,7 @@
 set -ex
 
 if [ "$SUPERSET_NO_DB_INIT" != "true" ]; then
-    source "/home/superset/docker-init.sh"
+    python bootstrap.py
```

Comment:
I didn't delete `docker-init.sh` in this pr. Most probably if we are ok with this change it will be better to revert `docker-init.sh` to original superset state to avoid conflicts there.